tensorflow/c/kernels/tensor_shape_utils.cc

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/c/kernels/tensor_shape_utils.h"
+
+#include <string>
+
+#include "tensorflow/c/tf_tensor.h"
+#include "tensorflow/core/platform/str_util.h"
```

Comment:
Is this include needed? Seems like you just use `StrAppend` which is from strcat.h.